apps/website/src/content/docs/skiptocontentlink.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: SkipToContentLink
+description: A skip-to-content link helps keyboard users quickly navigate to the main content.
+thumbnail: #TODO
+---
+
+<p>{frontmatter.description}</p>
+
+In some applications, numerous navigation links and other focusable elements can require a lot of keyboard presses before users reach their desired content. What is more, if users tab into a link that leads to another page with the same structure, they must tab through the same list of non-essential links again.
+
+While this may not be an issue for sighted or mouse users, it can be cumbersome for those who rely on keyboard navigation. Therefore, a skip link is crucial to improve accessibility and streamline navigation, designed for screen-reader and keyboard users.
+
+```jsx
+<SkipToContentLink href='#main-content-id' />
+```
```

Comment:
Do we need this code snippet here? Feels like it should be in the usage section.

```suggestion
```